### PR TITLE
Add icon for not result found in search

### PR DESCRIPTION
**Pull request by StaehliJ**
# Pull request

## Description

Self-explanatory

## Checklist

- [ ] Your branch has been rebased onto the `main` branch.
- [ ] APIs have been properly documented (if relevant).
- [ ] The documentation has been updated (if relevant).
- [ ] New unit tests have been written (if relevant).
- [ ] The demo has been updated (if relevant).
- [ ] All pull request status checks pass.

### CODE REVIEW

File: pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/ui/integrationLayer/SearchView.kt [hunk 1]
```diff
@@ -191,11 +192,21 @@ private fun SearchInput(
 private fun NoResults(
     modifier: Modifier = Modifier
 ) {
-    Box(
+    Column(
```
Comment (MGaetan89) [inline pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/ui/integrationLayer/SearchView.kt:195]: Maybe we could merge `NoResults` and `NoContent` together, and padd the `imageVector` and `text` as arguments?
Same in the other demo

**Review by MGaetan89 — APPROVED** [review]

File: pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/ui/integrationLayer/SearchView.kt (hunk 1 above)
Comment (StaehliJ) [reply pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/ui/integrationLayer/SearchView.kt:195]: I think the same.

File: pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/ui/integrationLayer/SearchView.kt [hunk 2]
```diff
@@ -187,36 +189,47 @@ private fun SearchInput(
...
         Icon(
-            imageVector = Icons.Default.Search,
+            imageVector = image,
             contentDescription = null,
             modifier = Modifier.size(56.dp)
         )
 
         Text(
-            text = stringResource(R.string.empty_search_query),
+            text = image,
```
Comment (MGaetan89) [inline pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/ui/integrationLayer/SearchView.kt:232]:
```suggestion
            text = message,
```

Comment (StaehliJ) [reply pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/ui/integrationLayer/SearchView.kt:232]: I was a bit fast on that one

**Review by MGaetan89 — APPROVED** [review]

**Pull request merged**